.gitignore

```diff
@@ -2,3 +2,4 @@ build
 composer.lock
 docs
 vendor
+.idea
```

Comment:
I prefer this to be added on global gitignore. ^_^ 
I also thumbs down on this PR on the framework but it still got merged. :))